app/routes.js

```diff
@@ -2,14 +2,30 @@ var express = require('express')
 const util = require('util')
 var getDatesInMonth = require('../lib/helper.js')
 
-var NotifyClient = require('notifications-node-client').NotifyClient
 
-var apiKey = process.env.NOTIFY_KEY
+// GET SECRETS
+var dbUser = process.env.DB_USER
+var dbPassword = process.env.DB_PASSWORD
+var notifyKey = process.env.NOTIFY_KEY
+var team_email = process.env.STANDARDS_TEAM_EMAIL
 
-var notifyClient = new NotifyClient(apiKey)
 
-var team_email = process.env.STANDARDS_TEAM_EMAIL
+// CONNECT TO DB
+var MongoClient = require('mongodb').MongoClient
+var dbURL = 'mongodb://' + dbUser + ':' + dbPassword + '@ds155028.mlab.com:55028/gds-assessments'
```

Comment:
It is probably worth: a) extracting the URL generation out into a function that takes an environment and returns the URL; b) allowing the host and database name to be configurable as well.